### PR TITLE
fix: fix to unoThemes issues

### PR DESCRIPTION
**Pull request by Arieldelossantos**
GitHub Issue: #

## Proposed Changes

 - [x] Bug fix
 - [ ] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## Description

- Fix canary issues relating to unothemes namespace change (https://github.com/unoplatform/Uno.Themes/blob/master/doc%2Fmaterial-migration.md#updating-to-unothemes-v50 )
- Changing the PR template

### Always applicable
No matter your changes, these checks always apply.
- [ ] Updated [CHANGELOG.md](../CHANGELOG.md).
  - Use the latest Major.Minor.X header if you do a **Patch** change.
  - Create a new Major.Minor.X header if you do a **Minor** or **Major** change.
  - If you create a new header, it aligns with the **Impact on version** section and matches what is generated in the build pipeline.
- [ ] Documentation files were updated according with the changes.
  - Update `README.md` and `TemplateConfig.md` if you made changes to **templating**.
  - Update `AzurePipelines.md` and `APP_README.md` if you made changes to **pipelines**.
  - Update `Diagnostics.md` if you made changes to **diagnostic tools**.
  - Update `Architecture.md` and its diagrams if you made **architecture decisions** or if you introduced new **recipes**.
  - ...and so forth: Make sure you update the documentation files associated to the recipes you changed. Review the topics by looking at the content of the `doc/` folder.
- [ ] Images about the template are referenced from the [wiki](https://github.com/nventive/UnoApplicationTemplate/wiki/Images) and added as images in this git.
- [ ] TODO comments are hints for next steps for users of the template and not planned work.

### Contextual
Based on your changes these checks may not apply.
- [ ] Automated tests for the changes have been added/updated.
- [ ] Tested on all relevant platforms

### CODE REVIEW

**Review by carlh98 — APPROVED** [review]

**Comment by jeanplevesque** [comment]
The PR template should probably be updated on this canary repo because it doesn't make sense.

**Review by Soap-141 — APPROVED** [review]

**Comment by Arieldelossantos** [comment]
should i merge this one and create another PR for the Template fix?? @jeanplevesque

File: src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj [hunk 1]
```diff
@@ -18,8 +18,8 @@
 	<ItemGroup>
```
Comment (jeanplevesque) [inline src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj:18]: I'm not sure `feat:` is the correct conventional commit type. It seems `fix:` would be more appropriate.

**Review by rfabre — APPROVED** [review]

**Comment by jeanplevesque** [comment]
> should i merge this one and create another PR for the Template fix?? @jeanplevesque

@Arieldelossantos Fix it in this PR so that you can also have a better PR template for this PR.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -4,9 +4,6 @@ GitHub Issue: #
 <!-- Please check one or more that apply to this PR. -->
```
Comment (jeanplevesque) [inline .github/PULL_REQUEST_TEMPLATE.md]: You should adjust this text to explain that this repository is the canary and therefore all PRs should be about fixing the canary and not be about adding new features.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -16,22 +13,10 @@ GitHub Issue: #
...
-  - New functionalities were added.
-- [ ] **Patch**
-  - A bug in behavior was fixed.
-  - Documentation was changed.
-
 ## PR Checklist 
 
 ### Always applicable
 No matter your changes, these checks always apply.
-- [ ] Your conventional commits are aligned with the **Impact on version** section.
```
Comment (jeanplevesque) [inline .github/PULL_REQUEST_TEMPLATE.md]: Keep this check

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -16,22 +13,10 @@ GitHub Issue: #
 <!-- (Please describe the changes that this PR introduces.) -->
 
 
-## Impact on version
```
Comment (jeanplevesque) [inline .github/PULL_REQUEST_TEMPLATE.md:19]: Keep this section, but add some more explanation:

## Impact on version
What would be the impact on the version if these changes would be merged back into the main repository?

Comment (Arieldelossantos) [reply .github/PULL_REQUEST_TEMPLATE.md:19]: done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (Arieldelossantos) [reply .github/PULL_REQUEST_TEMPLATE.md]: done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Arieldelossantos) [reply .github/PULL_REQUEST_TEMPLATE.md]: updated

**Review by jeanplevesque — APPROVED** [review]

**Comment by jeanplevesque** [comment]
These are wrong, no? I'm not sure to understand how you can have both these pipelines in the PR validation?
image

**Comment by Arieldelossantos** [comment]
> These are wrong, no? I'm not sure to understand how you can have both these pipelines in the PR validation? image

we're investigating why the variable is not restricting this validation, because we have one to check whether is a PR or not :/

**Pull request merged**